ERCS/erc-7498.md

```diff
@@ -0,0 +1,310 @@
+---
+eip: 7498
+title: NFT Redeemables
+description: Extension to ERC-721 and ERC-1155 for onchain and offchain redeemables
+author: Ryan Ghods (@ryanio), 0age (@0age), Adam Montgomery (@montasaurus), Stephan Min (@stephankmin)
+discussions-to: https://ethereum-magicians.org/t/erc-7498-nft-redeemables/15485
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-28
+requires: 165, 712, 721, 1155, 1271
+---
+
+## Abstract
+
+This specification introduces a new interface that extends [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) to enable the discovery and use of onchain and offchain redeemables for NFTs.
```

Comment:
I'd like to see a bit more technical information here. How does this interface enable discovery of redeemables? The abstract should be a terse technical summary of how the EIP works.

Reply:
updated in https://github.com/ethereum/ERCs/pull/27/commits/6cf49638075f962bf1c5196611d167a13e7e3e95, thanks!